core/trino-main/src/main/java/io/trino/execution/SqlTaskExecution.java

```diff
@@ -66,6 +66,7 @@
 import static com.google.common.base.Preconditions.checkState;
```

Comment:
Could you add the rationale for this change from the PR description to this commit's message as well ?